project.clj

```diff
@@ -1,12 +1,12 @@
-(defproject threatgrid/ductile "0.4.9-SNAPSHOT"
+(defproject threatgrid/ductile "0.5.0-SNAPSHOT"
   :description "Yet another Clojure client for Elasticsearch REST API, that fits our needs"
   :url "https://github.com/threatgrid/ductile"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.10.1"]
+  :dependencies [[org.clojure/clojure "1.12.0"]
```

Comment:
Why do you want to bump clojure version? I don't see any 1.12 specific changes in this PR